### PR TITLE
Heading tag improvements 

### PR DESCRIPTION
**Pull request by tomas-stefano**
Add specs to heading_tag helper.
Add documentation to it to see how we can use heading_tag.

This is related to #211.

### CODE REVIEW

**Comment by benbarnett** [comment]
Nice :+1:

**Comment by tomas-stefano** [comment]
Can I merge this one? cc @alexwllms @benlovell

File: spec/lib/dough/helpers_spec.rb [hunk 1]
```diff
@@ -19,6 +22,45 @@ def render(text)
   let(:renderer) { double(:Renderer, render: render) }
   subject(:helper) { Foo.new(helper_name: :inset_block, renderer: renderer, text: 'foo') }
 
+  describe '#heading_tag' do
+    let(:render) { double(:render, render: '', empty?: false) }
+
+    context 'when passes a block' do
```
Comment (benlovell) [inline spec/lib/dough/helpers_spec.rb]: when passing a block

File: spec/lib/dough/helpers_spec.rb [hunk 2]
```diff
@@ -19,6 +22,45 @@ def render(text)
...
+        end
+      end
+
+      it 'returns the header with content in the block and the level' do
+        expect(helper).to receive(:content_tag).with('h3', 'Header', 'role' => :heading, 'aria-level' => 3)
+        heading_tag
+      end
+    end
+
+    context 'when passes different heading levels' do
```
Comment (benlovell) [inline spec/lib/dough/helpers_spec.rb]: when passing...

File: spec/lib/dough/helpers_spec.rb [hunk 3]
```diff
@@ -19,6 +22,45 @@ def render(text)
...
+        helper.heading_tag('Header', level: 2)
+      end
+
+      it 'returns the level passed in the options' do
+        expect(helper).to receive(:content_tag).with('h2', 'Header', 'role' => :heading, 'aria-level' => 2)
+        heading_tag
+      end
+    end
+
+    context 'when does not pass the heading level' do
```
Comment (benlovell) [inline spec/lib/dough/helpers_spec.rb]: when not passing...

Comment (tomas-stefano) [inline spec/lib/dough/helpers_spec.rb]: Thanks about the review. Definitely I need to improve my english, :\.

**Comment by tomas-stefano** [comment]
All english fixes made. :) Thanks again @benlovell

**Comment by benlovell** [comment]
Looks good to me. :+1:

**Comment by tomas-stefano** [comment]
Ok! Shipping! \o/

**Pull request merged**